.github/workflows/maven-ci.yml

```diff
@@ -1,6 +1,6 @@
 #
 # The MIT License
-# Copyright © 2014-2019 Ilkka Seppälä
+# Copyright © 2014-2020 Ilkka Seppälä
```

Comment:
This is automatically generated using the license maven plugin, so it doesn't make much sense to update these manually for individual files.

Reply:
Okay. That's great, did not notice this for yamls in hidden folder.
Shall I let it remain as 2020, or change it back to 2019. As you said this is totally unnecessary.